app/src/main/java/com/hamdan/forzenbook/data/database/LoginDao.kt

```diff
@@ -17,4 +17,13 @@ interface LoginDao {
         """
     )
     suspend fun getToken(): LoginEntity?
+
+    // TODO When implementing further points for the token usage
```

Comment:
Where ticket number?

Reply:
A ticket for it doesn't exist yet, so I guess the ToDo can just be removed.